pandas/tests/arithmetic/conftest.py

```diff
@@ -190,7 +190,12 @@ def box(request):
 
 
 @pytest.fixture(
-    params=[pd.Index, pd.Series, pytest.param(pd.DataFrame, marks=pytest.mark.xfail)],
+    params=[
+        pd.Index,
+        pd.Series,
+        pytest.param(pd.DataFrame, marks=pytest.mark.xfail),
```

Comment:
Is there documentation as to why we `XFAIL` this?

Reply:
it's case-by-case in the places where it is used